authui/src/form.ts

```diff
@@ -194,6 +194,27 @@ export class RestoreFormController extends Controller {
   }
 }
 
+class MemoryStorage {
+  _storage: Record<string, string>;
+
+  constructor() {
+    this._storage = {};
+  }
+
+  setItem(key: string, value: string) {
+    this._storage[key] = value;
+  }
+
+  getItem(key: string): string {
+    return this._storage[key];
+  }
+
+  removeItem(key: string) {
+    delete this._storage[key];
+  }
+}
+const memoryStorage = new MemoryStorage();
```

Comment:
I found we actually have a thing called `RestoreFormController`.
It doesn't work because turbo didn't update the meta element in `<head>` in each navigation.